site/pages/api/create-consequence-network.api.ts

```diff
@@ -34,7 +36,12 @@ const createConsequenceNetwork = async (req: NextApiRequest, res: NextApiRespons
                 res,
             );
 
-            redirectTo(res, `${CREATE_CONSEQUENCE_NETWORK_PATH}/${body.disruptionId}/${body.consequenceIndex}`);
+            queryParam
+                ? redirectTo(
+                      res,
+                      `${CREATE_CONSEQUENCE_NETWORK_PATH}/${body.disruptionId}/${body.consequenceIndex}?${queryParam}`,
+                  )
+                : redirectTo(res, `${CREATE_CONSEQUENCE_NETWORK_PATH}/${body.disruptionId}/${body.consequenceIndex}`);
```

Comment:
These 2 redirects are exactly the same apart from if it has ?${queryParam} on the end. Instead of writing the same thing out twice, we can do this:

```suggestion
                redirectTo(res, `${CREATE_CONSEQUENCE_NETWORK_PATH}/${body.disruptionId}/${body.consequenceIndex}${queryParam ? `?${queryParam}` : ""}`)
```

Reply:
done